Main.java

```diff
@@ -9,7 +11,24 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+    // Creat a Scanner for user input
+    Scanner input = new Scanner(System.in);
+    
+    // Asks the users how and what they want to divide
+    System.out.println("Please enter in two integers, on seperate lines, to divide.");
+
+    // lets the users input numbers for the answers
+    int x1 = input.nextInt();
+    int x2 = input.nextInt();
+
+    // get the users answers
+    int answer = x1 % x2;
+    int answer2 = x1 / x2;
+
+    // tells the users the answer
+    System.out.println(answer2 + " with remainder of " + answer);
+
+    //Mr Lamont, I know im not supposed to comment here but I don't know how to do it other wise, I try adding (x1 + '/' + x2 + "is" answer 2 + " with remainder of " + answer) but it would not work
```

Comment:
No problems adding a little comment down there. To put the slash in, what you have is pretty close. Instead of single quotes, you can use normal quotation marks to make the slash a String. Also, don't forget the addition signs to link them all together. So it should look like
`
System.out.println(x1 + "/" + x2 + " is " + answer2 + " with a remainder of " + answer1);
`